crates/rome_js_formatter/src/jsx/attribute/attribute.rs

```diff
@@ -1,9 +1,18 @@
-use crate::{FormatElement, FormatResult, Formatter, ToFormatElement};
-use rome_js_syntax::JsxAttribute;
-use rome_rowan::AstNode;
+use crate::formatter_traits::FormatTokenAndNode;
+use crate::{format_elements, FormatElement, FormatResult, Formatter, ToFormatElement};
+use rome_js_syntax::{JsxAttribute, JsxAttributeFields};
 
 impl ToFormatElement for JsxAttribute {
     fn to_format_element(&self, formatter: &Formatter) -> FormatResult<FormatElement> {
-        Ok(formatter.format_verbatim(self.syntax()))
+        let JsxAttributeFields { name, initializer } = self.as_fields();
+
+        if let Some(initializer) = initializer {
+            Ok(format_elements![
+                name.format(formatter)?,
+                initializer.format(formatter)?
+            ])
+        } else {
+            Ok(format_elements![name.format(formatter)?])
+        }
```

Comment:
You should be able to write this as
```suggestion
        Ok(format_elements![
          name.format(formatter)?,
          initializer.format_or_empty(formatter)?
        ])
```